.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## PR Type
+What kind of change does this PR introduce?
+
```

Comment:
Add comment about adding a link to the issue it addresses. We really ought to require that all bug fixes and feature enhancements has an issue number associated with it.

Reply:
Added the link at the top of the template